codec-http/src/main/java/io/netty/handler/codec/rtsp/RtspHeaderNames.java

```diff
@@ -22,186 +22,234 @@
 /**
  * Standard RTSP header names.
  * <p>
- * These are all defined as lowercase to support HTTP/2 requirements while also not
- * violating RTSP/1.x requirements.  New header names should always be lowercase.
+ * These are all defined as lowercase to support HTTP/2 requirements while also
```

Comment:
Please revert the changes in this class as these are not really are needed for the functionality